django_ckeditor_5/package.json

```diff
@@ -42,7 +42,8 @@
     "@ckeditor/ckeditor5-theme-lark": "^41.1.0",
     "@ckeditor/ckeditor5-ui": "^41.1.0",
     "@ckeditor/ckeditor5-upload": "^41.1.0",
-    "@ckeditor/ckeditor5-word-count": "^41.1.0"
+    "@ckeditor/ckeditor5-word-count": "^41.1.0",
+    "@liqd/ckeditor5-file-uploader": "github:liqd/ckeditor5-file-uploader#v0.0.2"
```

Comment:
Hi @goapunk 
I have this error when running the npm install command
```
npm ERR! > @liqd/ckeditor5-file-uploader@0.0.1 prepare
npm ERR! > npm run ts:build && ckeditor5-package-tools export-package-as-javascript
npm ERR! 
npm ERR! 
npm ERR! > @liqd/ckeditor5-file-uploader@0.0.1 ts:build
npm ERR! > tsc -p ./tsconfig.release.json
npm ERR! 
npm ERR! src/fileuploadui.ts(14,10): error TS2305: Module '"ckeditor5/src/upload"' has no exported member 'FileDialogButtonView'.
npm ERR! src/fileuploadui.ts(63,23): error TS7006: Parameter 'evt' implicitly has an 'any' type.

```

Reply:
@hvlads fixed, should work again